### ISSUE TITLE
sosreport does not show available plugins

### ISSUE THREAD

**Issue by fluxcap1** [sosreport/sos#2217]
I install sos version 4.0 on openSUSE_Leap_15_2 as per the instructions published in README.md but I can't enable plugins. The plugins are available in /usr/lib/python3.6/site-packages/sos/report/plugins but sosreport doesn't process them.

`[Thu Aug 27 04:44:52 root@flux /usr/lib/python3.6/site-packages/sos/report/plugins] 
# sos report -l

sosreport (version 4.0)

no valid plugins found
`
Please help.

**Comment by TurboTurtle** [comment]
Looks like this is due to `OpenSuSePolicy` trying to activate on the presence of `/etc/SuSE-release`, which appears to no longer exist.

There is an `/etc/SUSE-brand`, which looks to be similar at least? If we can get confirmation that this new file is a replacement for `/etc/SuSE-release` then this is just a matter of updating the policy enablement check.

**Comment by TurboTurtle** [comment]
Of course we could also interrogate `/etc/os-release`, which may be more in line with the work @BryanQuigley was working on earlier.

**Comment by apsis-inc** [comment]
> Looks like this is due to `OpenSuSePolicy` trying to activate on the presence of `/etc/SuSE-release`, which appears to no longer exist.
> 
> There is an `/etc/SUSE-brand`, which looks to be similar at least? If we can get confirmation that this new file is a replacement for `/etc/SuSE-release` then this is just a matter of updating the policy enablement check.

Yeah, that seems to be the case. /etc/os-release is present though.

**Comment by fluxcap1** [comment]
> Looks like this is due to `OpenSuSePolicy` trying to activate on the presence of `/etc/SuSE-release`, which appears to no longer exist.
> 
> There is an `/etc/SUSE-brand`, which looks to be similar at least? If we can get confirmation that this new file is a replacement for `/etc/SuSE-release` then this is just a matter of updating the policy enablement check.

/etc/SuSE-release is abandoned for some years now. The configurations posted by my buddy apsis-inc are available. How do I fix this? Please help.

**Comment by TurboTurtle** [comment]
For the moment we can update the policy to check for `/etc/SUSE-brand` instead of `/etc/SuSE-release`, since this is a trivial fix with a major problem we can prioritize this. If you need it immediately, try making the following change in `sos/policies/suse.py`

```diff
diff --git a/sos/policies/suse.py b/sos/policies/suse.py
index 9418ead4..f6b3d3c8 100644
--- a/sos/policies/suse.py
+++ b/sos/policies/suse.py
@@ -110,4 +110,4 @@ No changes will be made to system configuration.
         if remote:
             return cls.distro in remote
 
-        return (os.path.isfile('/etc/SuSE-release'))
+        return (os.path.isfile('/etc/SUSE-brand'))
```

A longer term fix to survive vendor file changes like this is in the works, but we don't have an ETA on it.

**Comment by fluxcap1** [comment]
> For the moment we can update the policy to check for `/etc/SUSE-brand` instead of `/etc/SuSE-release`, since this is a trivial fix with a major problem we can prioritize this. If you need it immediately, try making the following change in `sos/policies/suse.py`
> 
> ```diff
> diff --git a/sos/policies/suse.py b/sos/policies/suse.py
> index 9418ead4..f6b3d3c8 100644
> --- a/sos/policies/suse.py
> +++ b/sos/policies/suse.py
> @@ -110,4 +110,4 @@ No changes will be made to system configuration.
>          if remote:
>              return cls.distro in remote
>  
> -        return (os.path.isfile('/etc/SuSE-release'))
> +        return (os.path.isfile('/etc/SUSE-brand'))
> ```
> 
> A longer term fix to survive vendor file changes like this is in the works, but we don't have an ETA on it.

I tried this last night but it didn't work.

**Comment by TurboTurtle** [comment]
Yeah, I loaded an OpenSUSE 15 VM and it appears there is another issue with the policy. A fix I've tested is posted in #2219 if you want to try that out.

**Comment by fluxcap1** [comment]
> Yeah, I loaded an OpenSUSE 15 VM and it appears there is another issue with the policy. A fix I've tested is posted in #2219 if you want to try that out.

Forked your repo, will report back in the evening. Thank you!

**Comment by fluxcap1** [comment]
I've managed to compile & install @TurboTurtle sos. There are no errors right now.
